lib/elements/confirm.js

```diff
@@ -19,7 +19,7 @@ class ConfirmPrompt extends Prompt {
   }
 
   reset() {
-    this.value = this.initialValue;
+    this.value = this.initialValue !== void 0 ? this.initialValue : false;
```

Comment:
Sorry 🙈 The `initialValue` here will never be `undefined` because we're `!!opts.initial` above. It will always be a boolean. Instead this can be:

```js
this.value = this.initialValue; // already true or false
```

Reply:
Lol such spaghetti code. Thanks

Reply:
Lol it's hard when it's spread out over a couple days. No biggy